overlord.pl

```diff
@@ -525,6 +537,12 @@ sub handleClient
         return;
     }
 
+    if ($header =~ m@^GET\s+\S*/finishRockHeaderUpdating\s@s) {
+        &finishRockHeaderUpdating();
```

Comment:
Please refactor to mimic what waitActiveRequests does, adding support for awaiting termination of all jobs matching the given (via request options) AsyncJob::typeName regex.

```suggestion
    if ($header =~ m@^GET\s+\S*/finishJobs\s@s) {
        my %options = &parseOptions($header);
        &finishJobs(\%options);
```

Reply:
Done.